tests/dicom/sort/test_sort_utils.py

```
@@ -0,0 +1,161 @@
+import pytest
+from pathlib import Path
+from pydicom.errors import InvalidDicomError
```

Ruff

`pydicom.errors.InvalidDicomError` imported but unused

Remove unused import: `pydicom.errors.InvalidDicomError`

(F401)